ucevcc/api.go

```diff
@@ -58,7 +58,7 @@ type UCEVCCInterface interface {
 	//
 	// parameters:
 	//   - entity: the entity of the EV
-	ManufacturerData(entity spineapi.EntityRemoteInterface) (string, string, error)
+	ManufacturerData(entity spineapi.EntityRemoteInterface) (*api.ManufacturerData, error)
```

Comment:
I do not use pointers in returns in the other APIs. Not sure if it is better the one or the other way. What do you think? See uccevc/api.go as an example

Reply:
Pointers are the go-ish way afaik. On the other hand side, return by value is cheap in Go and not returning pointers does allow to use the result without error checking, e.g. for chained returns. Ymmv.

Reply:
I'm normally a pointer guy. However in this cases one could argue return by value is better. 

- Other API returns are by value. Keeping the interfaces similar probably improves "user experience".
- As it is only a small struct one could probably even gain performance by using return by value.
- Makes the interface appear more complete, as no more changes to the structure are expected.

Will change the return type to by value